packages/network_integration/tests/testnet_integration.rs

```diff
@@ -206,6 +207,10 @@ fn run_airdrop() -> Result<()> {
     let initial_proof = proof_from_tree(&vec![0,1], &merlke_tree.layers());
 
     let a_permit = create_signed_permit(a_address_proof, ACCOUNT_KEY);
+    let account_permit = create_signed_permit(
+        AccountPermitMsg{
+            contract: HumanAddr(airdrop.address.clone()),
```

Comment:
if `airdrop.address` is a `HumanAddr` couldn't this be:
```suggestion
            contract: airdrop.address.clone(),
```

Reply:
Its a string, most of the CLI lin types try to keep it simple by using strings